convert.py

```diff
@@ -29,10 +30,20 @@
     default="", 
     help="The filepath of the .epub file to convert"
 )
+parser.add_argument("--offline", 

```

Comment:
I know this is not the most convenient change to make, but I'm not sure if "offline" is the right term here, it seems a little misleading

Maybe --local ?

Offline may give the impression that this functionality is meant to be run when the computer is offline (e.g. a fallback when there is no internet connection), as opposed to the fact that it runs Speech Synthesis locally

Reply:
No I think that's a good point I struggled with coming up with the right word here, and it was a pretty easy find & replace.